internal/artifacts/module_arch.go

```diff
@@ -32,19 +32,45 @@ func ExecuteBuild(source, target, desc, moduleName, platform string, wdGetter fu
 
 // ExecutePack - executes packing of module
 func ExecutePack(source, target, desc, moduleName, platform string, wdGetter func() (string, error)) error {
+
 	logs.Logger.Infof(`packing the "%v" module...`, moduleName)
 
 	loc, err := dir.Location(source, target, desc, wdGetter)
 	if err != nil {
 		return errors.Wrapf(err, `packing of the "%v" module failed when initializing the location`, moduleName)
 	}
 
-	module, _, err := commands.GetModuleAndCommands(loc, moduleName)
+	module, _, err := commands.GetModuleAndCommands(loc, source, moduleName)
```

Comment:
please don't use GetModuleAndCommands. if we fail on this it's not good to receive message about commands during normal pack. To receive module info use parse